openmdao/core/problem.py

```diff
@@ -137,9 +143,16 @@ class Problem(object):
         The number of times run_driver or run_model has been called.
     _warned : bool
         Bool to check if `value` deprecation warning has occured yet
+    _reports : str, bool, None
+        If _UNDEFINED, the OPENMDAO_REPORTS variable is used. Defaults to _UNDEFINED.
+        If given, reports should override OPENMDAO_REPORTS. If boolean, enable/disable all reports.
+        Since none is acceptable in the environment variable, a value of reports=None
+        is equivalent to reports=False. Otherwise, reports may be a sequence of
+        strings giving the names of the reports to run.
     """
 
-    def __init__(self, model=None, driver=None, comm=None, name=None, **options):
+    def __init__(self, model=None, driver=None, comm=None, name=None,
+                 reports=_UNDEFINED, **options):
```

Comment:
Add `reports_dir` here with similar behavior to `reports`, such that it defaults to _UNDEFINED and overrides the default/environment variable if provided.